.changeset/five-tigers-double.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
+---
+
+Add next-env.d.ts to .gitignore
```

Comment:
Should probably also delete the file https://github.com/t3-oss/create-t3-app/blob/next/cli/template/base/next-env.d.ts

Reply:
I would keep it to be consistent with `create-next-app`. It does copy this file into created app directory.

Reply:
Yeah I agree we shouldn't delete